waltz-tools/src/main/java/com/wepay/waltz/tools/UtilsCli.java

```diff
@@ -0,0 +1,28 @@
+package com.wepay.waltz.tools;
```

Comment:
Actually, can we rename this file as `CliUtils` instead of `UtilsCli`. Otherwise, it may not be understood as a utility class and might be mistaken as some other Cli.